docs/man_pages/project/testing/publish-ios.md

```diff
@@ -0,0 +1,50 @@
+publish ios
+==========
+
+<% if((isConsole && isMacOS) || isHtml) { %>
+Usage | Synopsis
+---|---
+General | `$ tns publish ios [<Application> [<Username> [<Password> [<Mobile Provisioning Profile Identifier> [<Code Sign Identity>]]]]]`
+
+Builds the project and uploads it to iTunesConnect. `Application Name`, `Username` and `Password` parameters are required but if not provided, the CLI will ask for them.
+
+### Attributes
+`<Application>` the name or Apple ID of the application created at itunesconnect.apple.com, or the index of the application as listed by `$ tns appstore`
+`<Username>` the Username for authentiation with iTunesConnect
+`<Password>` the Password for authentiation with iTunesConnect
+`<Mobile Provisioning Profile Identifier>` the identifier of the mobile provision which will be used for building. This can easily be acquired through the iPhone Configuration Utility
+`<Code Sign Identity>` the Code Sign Identity which will be used for building. It can be set to something generic like 'iPhone Distribution' and allow the build to automatically detect a Code Sign Identity.
```

Comment:
* No need for caps in Code Sign Identity
* It can be set to something generic like 'iPhone Distribution' and allow the build to automatically detect a Code Sign Identity. -> You can set it to something generic like ... to let the build automatically detect ...